build/ci/official.yml

```diff
@@ -86,8 +85,9 @@ steps:
       '**/*.pdb
       **/*.dll
       **/*.exe'
+    indexSources: false
```

Comment:
Any reason we don't want to index sources? Or is this something we want to do once per release, rather than once per build?

Reply:
Yes, I'm looking into moving these steps into the release pipeline instead. Optimally this yml will eventually only include build, tests, and signing and then the release will take care of everything else.